In [19]:
import numpy as np
import statistics
from scipy.stats import variation 

# Section 3.1 

###### Question 1

In [1]:
set = [3, 11, 5, 5, 6, 7, 8, 8, 8, 9, 10, 10]

In [6]:
#a. Mean
statistics.mean(set)

7.5

In [7]:
#b. Median
statistics.median(set)

8.0

In [8]:
#c. mode
statistics.mode(set)

8

###### Question 2

In [9]:
midterm = 83
final_exam = 95

In [11]:
0.4*midterm + 0.6*final_exam

90.2

Yes, the weighted average is 90.2 which is an A.

###### Question 3

# Section 3.2

###### Question 1

In [12]:
mileage = [3, 11, 5, 5, 6, 7, 8, 8, 8, 9, 10, 10]

In [13]:
#a.Mean
statistics.mean(mileage)

7.5

In [14]:
#Standard Deviation
statistics.stdev(mileage)

2.393172105652397

In [16]:
#1st standard deviation 68%
a = statistics.mean(mileage) - statistics.stdev(mileage)
b = statistics.mean(mileage) + statistics.stdev(mileage)
[a,b]

[5.106827894347603, 9.893172105652397]

In [17]:
#2nd standard deviation 95%
c = statistics.mean(mileage) - 2*statistics.stdev(mileage)
d = statistics.mean(mileage) + 2*statistics.stdev(mileage)
[c,d]

[2.713655788695206, 12.286344211304794]

In [18]:
#3rd standard deviation 99.7%
e = statistics.mean(mileage) - 3*statistics.stdev(mileage)
f = statistics.mean(mileage) + 3*statistics.stdev(mileage)
[e,f]

[0.3204836830428093, 14.67951631695719]

In [ ]:
#68% of the data falls between 5.10 and 9.89
#95% of the data falls between 2.71 and 12.29
#99.7% of the data falls between 0.32 and 14.68

###### Question 2

In [21]:
#Compute Coefficient of Variation - cv
mean = 20
stdev = 2
cv = (stdev/mean)*100
cv

10.0

In [22]:
#Compute an 88.9% Chebyshev interval around the population mean.
a = mean - 3*stdev 
b  = mean + 3*stdev
[a,b]

[14, 26]

In [28]:
k = 1/np.sqrt(0.25)

In [29]:
mean1 = 15
stdev1 = 3

In [30]:
c = mean1 - k*stdev1 
d  = mean1 + k*stdev1
[c,d]

[9.0, 21.0]

In [31]:
#75% Chebyshev interval around the sample mean is [9, 21]

###### Question 4

In [32]:
#a.
#63 mph and 71 mph are 1 standard deviation from the mean. Since the distribution is bell-shaped and this is a normal distribution.
#The empirical rule tells us that about 68% of the data is within 1 standard deviation from the mean.

In [ ]:
#b.
#99.7% of the data points should lie between 3 standard deviations of the mean on a normally distributed data set. The point   

In [34]:
def multiply(a,b):
    print(a*b)

In [35]:
multiply(3,7)

21


In [ ]:
def duplicate_count(text):
      characters = text.split('')
      #Create a loop that loops through each character in the string
      for i in range (0,len(text)):
          #accept the string text, append the string into lower case
            if 

In [37]:
TEXT = "UPPER"

In [48]:
a = TEXT.lower().split('')

ValueError: empty separator

In [49]:
a[0]

'upper'